v2_bridges/erc20_on_int/src/int/eth_tx_info.rs

```diff
@@ -22,6 +22,7 @@ pub struct Erc20OnIntEthTxInfo {
     pub origin_chain_id: EthChainId,
     pub token_recipient: EthAddress,
     pub destination_address: String,
+    pub(crate) metadata_event: Option<PTokensRouterMetadataEvent>,
```

Comment:
Just curious - why did you put this here and not at the bottom of the struct?

Reply:
I think that it was probably an accident, changing!